bin/v-import-cpanel

```diff
@@ -31,16 +31,9 @@ if [ ! -e /usr/bin/rsync ] || [ ! -e /usr/bin/file ]; then
 	echo "rsync not installed, try install it"
 	echo "This script need: rsync, file"
 	echo "#######################################"
-	if [ -e /etc/redhat-release ]; then
-		echo "Run: yum install rsync file"
-	else
-		echo "Run: apt-get install rsync file"
-	fi
+    echo "Run: apt-get install rsync file"
```

Comment:
Why was this commented out?

Reply:
Not remember now, maybe because hestia not support centos

but I make new changes to this script in https://github.com/Skamasle/sk-import-cpanel-backup-to-vestacp/blob/master/v-import-cpanel-v2